modules/gapi/samples/face_detection.cpp

```diff
@@ -613,24 +621,26 @@ int main(int argc, char* argv[]) {
     in_resized[0] = cv::gapi::resize(in_originalRGB, level_size[0]);
     in_transposed[0] = custom::Transpose::on(in_resized[0]);
     std::tie(regressions[0], scores[0]) = run_mtcnn_p(in_transposed[0], get_pnet_level_name(level_size[0]));
-    cv::GArray<custom::Face> faces0 = custom::BuildFaces::on(scores[0], regressions[0], scales[0], conf_thresh_p);
-    nms_p_faces[0] = custom::RunNMS::on(faces0, 0.5, false);
+    cv::GArray<custom::Face> faces0 = custom::BuildFaces::on(scores[0], regressions[0], static_cast<float>(scales[0]), conf_thresh_p);
```

Comment:
Is it necessary to use `double` for scales?

Reply:
Yes. Otherwise pyramid level sizes are differ +-1 comparing to the reference OMZ and results can't be comparable anymore.

Reply:
Ok